File: dev-docs/snippets/groups/create.swift
import JsBaoClient

// Create a new group. `groupType`, `groupId`, and `name` are required.
func create(client: JsBaoClient) async throws {
  // #region example
  let group = try await client.groups.create(params: CreateGroupParams(
    groupType: "team",
    groupId: "design",
    name: "Design Team",
    description: "Product design crew"
  ))
  // #endregion example
  _ = group
}

File: dev-docs/snippets/collections/create.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Create a new collection. `name` is required; `collectionType`/`contextId`
// are immutable after create.
export async function create(client: JsBaoClient) {
  // #region example
  const collection = await client.collections.create({
    name: "Q3 Planning",
    description: "Docs for the Q3 planning cycle",
    collectionType: "default",
  });
  // #endregion example
  return collection;
}
